src/test/java/seedu/guest/model/guest/NumberOfGuestsTest.java

```diff
@@ -36,5 +38,12 @@ public void isValidNumberOfGuests() {
         assertTrue(NumberOfGuests.isValidNumberOfGuests("3"));
         assertTrue(NumberOfGuests.isValidNumberOfGuests("4"));
     }
+
+    @Test
+    public void hashcode() {
```

Comment:
Similarly, should we standardise the format for these tests with line spacing and comments?

Reply:
Agreed. Added line spacing and comments.